src/css/footer.css

```diff
@@ -0,0 +1,68 @@
+.tagline-logo {
+  font-family: 'Impact', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 100%;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+}
+
+.logo-web {
+  color: var(--accent-green);
+}
+
+.footer {
+  padding-top: 40px;
+  padding-bottom: 24px;
+  line-height: 150%;
+  letter-spacing: 0.03em;
+  color: var(--black);
+}
+
+.footer-tumb {
+  display: block;
```

Comment:
this property is not necessary for .footer-tumb. Because div have property `dislpay: block` as default